IsraelHiking.Web/src/scss/common.scss

```diff
@@ -98,19 +98,29 @@ body {
         padding: 0 !important;
         display: block;
         width: v.$app-button-size;
-        height: v.$app-button-size;
+        height: 44px;
 
         & i {
             vertical-align: middle !important;
         }
     }
+    .active:not(.active:first-child , .search .active ){
```

Comment:
The selector `.active:not(.active:first-child, .search .active) `targets buttons with the active class, but excludes those that are the first child of their parent(first of both vertical sidebars) or within a `.search` bar section. This way, it applies margin only to the `.active` button with a green border  and excludes that match in `.not()`  conditions.

Added this code to avoid overlapping of green border of buttons

Reply:
Is there a simpler way to achieve this?

Reply:
I don’t think we can use a simple selector with CSS alone because it applies styles conditionally. Adding a margin without this condition disrupts the alignment of the buttons.

Reply:
Can you create a video or animated gif to show if the buttons are moving when two buttons that are close are active? I don't understand where this is used and how it solves the problem...

Reply:
Also stuff that are specific only to a single screen should be places in the scss of the relevant component, search.component in this case.